endpoints/citrus-http/src/test/java/com/consol/citrus/http/yaml/HttpClientTest.java

```diff
@@ -45,6 +45,7 @@
 import com.consol.citrus.message.MessageHeaders;
 import com.consol.citrus.message.MessageQueue;
 import com.consol.citrus.spi.BindToRegistry;
+import com.consol.citrus.util.SocketUtils;
```

Comment:
as pointed out, I would probably not do this. we will immediately have "legacy code" in the codebase..